README.md

```diff
@@ -10,8 +10,11 @@ Before running you will have to set up an account at https://loggly.com and add
 
 Role Variables
 --------------
+Store the following in [a vault file](http://docs.ansible.com/ansible/playbooks_vault.html):
 
-LOGGLY TOKEN: The token associated with your loggly account
+    secrets:
```

Comment:
Maybe using Github's code directives for markdown would be good here. I know it's just a read and not super critical.

```yaml
secrets:
  loggly:
    token: yourlogglytokenhere
```

Use the markdown: '```yaml' to open and '```' to close the block.